src/main/java/smartthings/ratpack/sqs/DefaultSqsService.java

```diff
@@ -1,49 +1,52 @@
 package smartthings.ratpack.sqs;
 
-import com.amazonaws.services.sqs.AmazonSQS;
-import com.amazonaws.services.sqs.model.*;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 import ratpack.exec.Blocking;
 import ratpack.exec.Promise;
 import ratpack.service.Service;
 import ratpack.service.StopEvent;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import software.amazon.awssdk.services.sqs.model.*;
 
 /**
  * Default implementation for communicating with AWS SQS.
  */
 @Singleton
 public class DefaultSqsService implements SqsService, Service {
 
-    private final AmazonSQS sqs;
+    private final SqsClient sqs;
```

Comment:
Note - the v2 aws lib has an async sqs client (think it's netty based).  It works pretty seemlessly with Ratpack without utilizing blocking threads.  That said - it's a bigger lift then what you have here, and what you have is functionally equivalent.